### PR TITLE
Add doc examples for `uuid::Uuid::as_bytes`.

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -371,6 +371,20 @@ impl Uuid {
     }
 
     /// Return an array of 16 octets containing the UUID data
```
Comment (GuillaumeGomez) [inline src/lib.rs]: I just saw: we say octet in english? Isn't it byte?

**Comment by GuillaumeGomez** [comment]
Seems good for me.

**Comment by alexcrichton** [comment]
Thanks!

**Pull request merged**